lib/commons/text/accessible-text.js

```diff
@@ -137,34 +138,34 @@ function nonEmptyText(t) {
  * @return {string}
  */
 text.accessibleText = function(element, inLabelledByContext) {
-	//todo: implement shadowDOM
-	var accessibleNameComputation;
-	var encounteredNodes = [];
+	let accessibleNameComputation;
+	const encounteredNodes = [];
+	if (element instanceof Node) {
```

Comment:
This works differently from a lot of the other commons - which do not try to maintain backwards compatibility. Whether we want to support backwards compatibility or not and if we do, we need to change those other utilities.